src/core.js

```diff
@@ -403,11 +390,17 @@ define([
             });
         });
 
+        // Section toggling events: within section changer and when clicking on section header
```

Comment:
I misunderstood why we were doing this and didn't implement it right. Have now changed the section headers to no longer be clickable, and now when you hide a section, it removes both the section and the header. This makes the basic experience, where you're only using the basic section, more manageable (you don't see the headers for Logic, etc. and therefore don't   click on them and get distracted). We're planning to user test this next week to see if it's helpful.